scene/gui/code_edit.cpp

```diff
@@ -1525,7 +1525,10 @@ void CodeEdit::_clear_delimiters(DelimiterType p_type) {
 			delimiters.remove(i);
 		}
 	}
-	delimiter_cache.clear();
+	if (!setting_delimiters) {
+		delimiter_cache.clear();
```

Comment:
Apologises, should have been more clear, still need to call `delimiter_cache.clear()`, only need to skip `_update_delimiter_cache` else the cache will grow quite large.

Reply:
That did come across my mind when I was doing the change, I should have asked 🤣